CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# Contributing
```

Comment:
I'm thinking we could have columns like these:

```
| org name | project name | author |
```

Then sort according to org name since people will be more likely to search by that. However, I am not sure where to link the writeup. Project name? Since each writeup is actually based on a project? What do you think?